app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -550,7 +571,14 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 													subTitle = course.title;
 													var plannedSeats = sectionGroup.plannedSeats || "0";
 													plannedSeatsHtml = "<small>Seats: " + plannedSeats + "</small>";
-													unitsLow = "<small>Units: " + course.unitsLow + "</small>";
+
+													if (sectionGroup.unitsVariable) {
+														units = "Units (Variable): " + sectionGroup.unitsVariable;
+													} else if (course.unitsHigh) {
```

Comment:
We mess up this logic a lot, so I would instead add a transient helper on the entity in the backend so you just see a displayUnits string property which will be the correct thing to show